ports/libssh/portfile.cmake

```
@@ -19,10 +19,18 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     zlib    WITH_ZLIB
 )
 
+if (VCPKG_TARGET_IS_ANDROID)
+	set(EXTRA_ARGS "-DWITH_MBEDTLS=TRUE"
+			"-DWITH_SERVER=FALSE"
+			"-DWITH_PCAP=FALSE"
+			)
```

```suggestion
	set(EXTRA_ARGS "-DWITH_SERVER=FALSE"
			"-DWITH_PCAP=FALSE"
			)
```

`vcpkg_check_features()` will already add `-DWITH_MBEDTLS` based on the feature setting.

---

Fixed